src/app.types.tsx

```diff
@@ -80,6 +80,7 @@ export interface Manufacturer extends ManufacturerDetails {
 
 export interface AllowedValuesList {
   type: 'list';
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   values: any[];
```

Comment:
Good point, although I don't want to touch it here as it was `any` before so this just disables the warning for now, I suspect it would need changes elsewhere otherwise.